authors/apps/comments/views.py

```diff
@@ -0,0 +1,105 @@
+from django.shortcuts import get_object_or_404
+from rest_framework.response import Response
+from rest_framework.serializers import ValidationError
+from rest_framework import status
+from rest_framework import generics, serializers, status
+from rest_framework.permissions import IsAuthenticated
+
+from authors.apps.articles.models import Article
+from ..profiles.models import Profile
+
+from .models import Comment
+from .renderers import CommentJSONRenderer
+from .serializers import CommentSerializer
+
+
+class CommentCreateListView(generics.ListCreateAPIView):
+    """
+
+    create comments and retrieve comments 
+
+    """
+    serializer_class = CommentSerializer
+    permission_classes = (IsAuthenticated, )
+    renderer_classes = (CommentJSONRenderer, )
+    queryset = Comment.objects.all()
+    lookup_field = 'article'
+
+    def post(self, request, slug):
+        """
+        This method posts a comment to an article
+        """
+        article = get_object_or_404(Article, slug=slug)
+        data = request.data.get('comment')
+        author = Profile.objects.get(user=request.user)
+
+        serializer = self.serializer_class(data=data, context={'article': article})
+        serializer.is_valid(raise_exception=True)
+        serializer.save(commented_by=author, article=article)
+        return Response({"comment": serializer.data,
+            "message":"comment successfully added"},
+            status=status.HTTP_201_CREATED)
+
+    def get(self, request, slug):
+
+        article = get_object_or_404(Article, slug=slug)
+        comment = Comment.objects.filter(article=article)
+        serializer = self.serializer_class(comment, many=True, context={"request":request})
+        return Response(serializer.data, status=status.HTTP_200_OK)
```

Comment:
Thanks for the good work @wasibani-roy, However when getting a list of comments, we could have the key as comments and not comments since it represents a list of more than one items.
```
 {
"comments:[]

} 
```
not 
```
 {
"comment":[]
} 
```